deploy.php

```diff
@@ -55,10 +55,11 @@
     'deploy:shared',
     // build and upload assets
     'app:assets:deploy',
-    'app:graphql:deploy',
     // copy vendors folder between releases before running composer install to speed it up
     'deploy:copy_dirs',
     'deploy:vendors',
+    // build admin schema
+    'app:graphql:deploy',
```

Comment:
We don't need to call this `app:graphql:deploy` any more. Just use `app:graphql:generate` directly and remove this `deploy` group altogether.

Reply:
Like so then? [3370c59](https://github.com/netgen/media-site/pull/224/commits/3370c59ff1a17d5d54d3a17efff383a6177297f1)